### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by mik-laj**

### CODE REVIEW

File: packages/mjml-section/src/index.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ class Section extends Component {
   renderSection () {
     const { renderWrappedOutlookChildren, mjAttribute, children, parentWidth } = this.props
     const fullWidth = this.isFullWidth()
-    const divProps = fullWidth ? {} : {
+    const divProps = {
```
Comment (iRyusa) [inline packages/mjml-section/src/index.js]: Hi, sorry this is needed to avoid duplicate css-class when the section is in full width

**Comment by mik-laj** [comment]
Look at method `render`. It's do similar check.

**Comment by iRyusa** [comment]
https://github.com/mjmlio/mjml/pull/833/files/800501863f254c14a8b8f3dfe54176808bce6977#diff-327a862c444f85fc9611b8b91e6519b1L181
renderFullWidthSection calls renderSection, so that's why you still need this

**Comment by mik-laj** [comment]
My mistake. Sorry for taking the time.

**Pull request closed without merging**